release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2643,3 +2643,17 @@ link:https://access.redhat.com/solutions/6324771[{product-title} 4.8.11 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+// TODO: Add this to the end of the appropriate 4.8 z-stream RNs that this check is added with.
+// TODO: Do we need to include this notice in all future z-stream RN as well?
+[id="ocp-4-8-preparing-upgrade-next-release"]
+==== Preparing to upgrade to the next {product-title} release
+
+// TODO: replace "TODO" with appropriate z-stream, and is "check" the correct term?
+{product-title} 4.8.TODO introduced a check that will impact upgrading to the next {product-title} release, which is currently planned to be {product-title} 4.9. This is because Kubernetes 1.22, which {product-title} 4.9 is expected to use, removed a significant number of deprecated `v1beta1` APIs.
+
+This check requires an administrator to provide a manual acknowledgment before the cluster can be upgraded from {product-title} 4.8 to 4.9. This is to help prevent issues after upgrading to {product-title} 4.9, where APIs that have been removed are still in use by the cluster. Administrators must evaluate their cluster for any removed APIs in use and migrate them to use the appropriate new API version. After this is done, the administrator can provide the admin acknowledgment.
+
+All clusters require this admin acknowledgment before they can be upgraded to {product-title} 4.9.
+
+For more information on the list of removed Kubernetes APIs, tips for how to evaluate your cluster for removed APIs in use, and how to provide the admin acknowledgment, see link:https://access.redhat.com/articles/6329921[Preparing to upgrade to OpenShift Container Platform 4.9].
```

Comment:
nit: does `{product-title}` not work here, or are we just guarding against the stable solution title vs. the various products that this `adoc` will be included in?

Reply:
Yeah, when we do an external link like this, where the title is hard coded, we usually use the hard-coded title instead of that {product-title} attribute.